static/speakers.html

```diff
@@ -122,7 +122,7 @@ <h1 class="logo" style="float: none;display:inline-block">
                 <li><a href="speakers.html">Speakers</a></li>
                 <li><a href="contact.html">Contact us</a></li>
               </ul>
-              <p class="copyright">© 2015 Rotaract Milimani. All rights reserved</p>
+              <p class="copyright">© 2015 Rotaract Milimani. All rights reserved. Built by <a href="kennethkinyanjui.info">kenju254</a></p>
```

Comment:
@kenju254 Don't know whether you've noticed but this is treated like a relative link
The link location shows up as https://linkkenya2015.appspot.com/kennethkinyanjui.info